types/discord-rpc/index.d.ts

```diff
@@ -199,4 +200,5 @@ export interface Presence {
     matchSecret?: string;
     spectateSecret?: string;
     joinSecret?: string;
+    buttons?: Array<{ label: string, url: string }>;
```

Comment:
```suggestion
    buttons?: {
        label: string;
        url: string;
    }[];
```
I think this looks a bit nicer

Reply:
The npm tests advised against using `[]` to declare an array and I tried following the code style of arrays, but I believe using `Array<{}>` is concise and easy to understand since there are only two internal types within the array object.